src/plane.js

```diff
@@ -0,0 +1,16 @@
+class Plane {
+
+    constructor(id) {
+        if (id === undefined) {
+            throw new Error(`This is not a valid Plane ID`);
```

Comment:
Throwing an Error created some issues with testing.
These issues have been documented, and highlighted by a comment, in the relevant files

Reply:
I researched how to get an Error message to appear (which lead me to using "throw new Error");.
I believe there might have been another way to do this which facilitated following the methods of TDD more closely, such as  giving an undefined ID a default value and making a plane with a default value ID run a different code (using if statements); however, throwing an error appeared to be the way to do it in the fewest lines of code.